com.musala.xml.parser/src/com/musala/xml/parser/Main.java

```diff
@@ -0,0 +1,132 @@
+package com.musala.xml.parser;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import com.musala.xml.parser.model.School;
+import com.musala.xml.parser.model.Student;
+import com.musala.xml.parser.model.Teacher;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException, JAXBException {
+
+        ClassLoader classload = Main.class.getClassLoader();
+        File configFile = new File(classload.getResource(Constants.XML_INTERNAL_FILE_PATH).getFile());
+        JAXBContext jaxb = JAXBContext.newInstance(School.class);
+        Unmarshaller jaxbUnmarsheller = jaxb.createUnmarshaller();
+        School mySchool = (School) jaxbUnmarsheller.unmarshal(configFile);
+        Print(mySchool);
+    }
+
+    private static void Print(School myschool) {
+
+        int numberOfClasses = myschool.getSchoolClasses().size();
+        System.out.printf("The school name is: %s!%n There %s %d school classes.%n", myschool.getName(),
```

Comment:
You should apply the single/plural conversion for the word "class/classes" as well (currently, running the program outputs "There is 1 school classes").